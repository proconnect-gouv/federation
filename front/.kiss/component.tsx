import React from 'react';

interface DumbComponentProps {
  className: string;
}

export const DumbComponent: React.FC<DumbComponentProps> = React.memo(
  ({ className }: DumbComponentProps) => {
    return <div />;
  },
);

DumbComponent.displayName = 'DumbComponent';
