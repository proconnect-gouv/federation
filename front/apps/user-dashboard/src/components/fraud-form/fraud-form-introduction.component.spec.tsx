import { render } from '@testing-library/react';

import { FraudFormIntroductionComponent } from './fraud-form-introduction.component';

describe('FraudFormIntroductionComponent', () => {
  it('should match the snapshot', () => {
    // When
    const { container } = render(<FraudFormIntroductionComponent />);

    // Then
    expect(container).toMatchSnapshot();
  });
});
