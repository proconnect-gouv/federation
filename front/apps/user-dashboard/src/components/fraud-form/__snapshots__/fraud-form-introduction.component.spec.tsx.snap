// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FraudFormIntroductionComponent should match the snapshot 1`] = `
<div>
  <div>
    <h1
      class="fr-h3 is-blue-france fr-mb-1w"
    >
      <b>
        Je signale une usurpation
      </b>
    </h1>
    <p>
      En remplissant ce formulaire, le support FranceConnect sera inform√© de votre demande. Nous vous recontacterons sous peu.
    </p>
  </div>
</div>
`;
